.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: objective-c
-osx_image: xcode9.2
+osx_image: xcode9.3beta
```

Comment:
Does this mean the change will only affect the beta version of Xcode 9.3? I'm on the real deal: Version 9.3 (9E145)